### ISSUE TITLE
PhoneNumberSource.update does not update

### ISSUE THREAD

**Issue by doerfli** [doerfli/leavemealone#41]
only checks if entry exists and if not creates a new one. if it exists, nothing happens.

**Comment by pamapa** [comment]
yes, it looks like this, but the PhoneNumberSource only has the ID and the name, if the name already exists, nothing needs to be made at all. For me this is not a bug.
